src/DesignKit.tsx

```diff
@@ -4200,7 +4201,7 @@ const DesignKit: React.FC<{
 
   return (
     // wrap in an antd component so links look correct
-    <UIProvider theme={theme} themeIsDark={themeIsDark}>
+    <UIProvider priority="low" theme={theme} themeIsDark={themeIsDark}>
```

Comment:
I'm curious why we have a UIProvider here in `DesignKit` and also below in `DesignKitContainer`